### PR TITLE
[web] Website Footer

### PR DESCRIPTION
**Pull request by JonathanXu1**
Ticket | Title
---|---
#24 | [web] add website footer
#22 | [web] add in a support method Task

## Description

- Added a website footer as per #24 and #22 
- Fixed bug where one specific FAQ question toggled two questions
- Added support for Font Awesome icons

## Screenshot
Screenshot from 2019-12-24 23-32-59

### CODE REVIEW

File: src/client/components/landing/LandingFooter.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            </Badge>
+          </a>
+          <a href="https://www.reddit.com/r/MyriadeMining/" target="_blank">
+            <Badge pill variant="dark" className="m-1">
+              <i class="fab fa-reddit"></i>
+            </Badge>
+          </a>
+        </Row>
+        <Row className="justify-content-md-center m-2">
+          <a className="m-1" target="_blank">Privacy Policy</a>
```
Comment (hanburger97) [inline src/client/components/landing/LandingFooter.jsx]: Should we create the target file for these?

File: src/client/styles/components/landing/footer.less [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.footer{
+    background-color: #F07C00;
```
Comment (hanburger97) [inline src/client/styles/components/landing/footer.less]: Also can we make the color to dark grey or black or navy blue? The orange might seem excessive given that the rest of the page will be orange

File: src/client/components/landing/LandingFooter.jsx (hunk 1 above)
Comment (JonathanXu1) [reply src/client/components/landing/LandingFooter.jsx]: Yep, I'll be creating pages for those in another PR

File: src/client/styles/components/landing/footer.less (hunk 2 above)
Comment (JonathanXu1) [reply src/client/styles/components/landing/footer.less]: Yeah sure, I'll post screenshots of some colors soon

Comment (JonathanXu1) [reply src/client/styles/components/landing/footer.less]: Screenshot from 2019-12-26 15-58-19
@hanburger97 how's this?

**Pull request merged**